translations/ja.json

```diff
@@ -454,57 +454,57 @@
-	"Please re-eval your docker-env, To ensure your environment variables have updated ports:\n\n\t'minikube -p {{.profile_name}} docker-env'\n\n\t": "",
-	"Please re-eval your podman-env, To ensure your environment variables have updated ports:\n\n\t'minikube -p {{.profile_name}} podman-env'\n\n\t": "",
-	"Please run `minikube logs --file=logs.txt` and attach logs.txt to the GitHub issue.": "",
-	"Please see {{.documentation_url}} for more details": "",
-	"Please specify the directory to be mounted: \n\tminikube mount \u003csource directory\u003e:\u003ctarget directory\u003e   (example: \"/host-home:/vm-home\")": "",
-	"Please specify the path to copy: \n\tminikube cp \u003csource file path\u003e \u003ctarget file absolute path\u003e (example: \"minikube cp a/b.txt /copied.txt\")": "",
-	"Please try purging minikube using `minikube delete --all --purge`": "",
-	"Please upgrade the '{{.driver_executable}}'. {{.documentation_url}}": "「{{.driver_executable}}」をアップグレードしてください。{{.documentation_url}}",
-	"Please visit the following link for documentation around this: \n\thttps://help.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-docker-for-use-with-github-packages#authenticating-to-github-packages\n": "",
-	"Populates the specified folder with documentation in markdown about minikube": "",
-	"PowerShell is running in constrained mode, which is incompatible with Hyper-V scripting.": "",
+	"Please install the minikube hyperkit VM driver, or select an alternative --driver": "minikube hyperkit VM ドライバーをインストールするか、--driver で別のドライバーを選択してください",
+	"Please install the minikube kvm2 VM driver, or select an alternative --driver": "minikube kvm2 VM ドライバーをインストールするか、--driver で別のドライバーを選択してください",
+	"Please make sure the service you are looking for is deployed or is in the correct namespace.": "探しているサービスがデプロイされている、あるいは正しいネームスペース中にあることを確認してください。",
+	"Please provide a path or url to build": "ビルドするパスまたは URL を指定してください",
+	"Please provide an image in the container runtime to save from minikube via \u003cminikube image save IMAGE_NAME\u003e": "\u003cminikube image save IMAGE_NAME\u003e で minikube からセーブする、コンテナーランタイム中のイメージを指定してください",
+	"Please provide an image in your local daemon to load into minikube via \u003cminikube image load IMAGE_NAME\u003e": "\u003cminikube image load IMAGE_NAME\u003e で minikube 中にロードする、ローカルデーモンが入ったイメージを指定してください",
+	"Please provide source and target image": "ソースイメージとターゲットイメージを指定してください",
+	"Please re-eval your docker-env, To ensure your environment variables have updated ports:\n\n\t'minikube -p {{.profile_name}} docker-env'\n\n\t": "環境変数が更新されたポート番号を持つことを確実にするために docker-env を再評価してください:\n\n\t'minikube -p {{.profile_name}} docker-env'\n\n\t",
+	"Please re-eval your podman-env, To ensure your environment variables have updated ports:\n\n\t'minikube -p {{.profile_name}} podman-env'\n\n\t": "環境変数が更新されたポート番号を持つことを確実にするために podman-env を再評価してください:\n\n\t'minikube -p {{.profile_name}} podman-env'\n\n\t",
```

Comment:
Maybe, "再適用" is easier to understand.

Reply:
I think "再適用" is better than other options, because `minikube docker-env`  [sets up docker env variables](https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/docker-env.go#L243) not confirmation("確認" in Japanese).

BTW, docker-machine env manual is at [here](https://docs.docker.jp/machine/reference/env.html).
I could not reach the page by click link yosshy noted above.

Reply:
I agree with "再適用".